### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by tsionyx**

### CODE REVIEW

File: src/infi/clickhouse_orm/utils.py [hunk 1]
```diff
@@ -48,29 +49,8 @@ def parse_array(array_string):
...
-                raise ValueError('Missing closing quote: "%s"' % array_string)
-            values.append(array_string[1 : match.start() + 1])
-            array_string = array_string[match.end():]
-        else:
-            # Start of non-quoted value, find its end
-            match = re.search(r",|\]", array_string)
-            values.append(array_string[1 : match.start() + 1])
-            array_string = array_string[match.end():]
+
+    return ast.literal_eval(array_string)
```
Comment (ishirav) [inline src/infi/clickhouse_orm/utils.py]: This is a very elegant way to parse array strings, however I'm not sure it's the right way. The original implementation returned only strings, and let the Field convert it to the correct Pythonic type. In your implementation, literal_eval already converts to various types. Furthermore, literal_eval supports types that we do not expect to receive, such as dict / boolean / tuple. What is your view on this matter?

File: src/infi/clickhouse_orm/fields.py [hunk 2]
```diff
@@ -107,6 +107,8 @@ def to_python(self, value):
         if isinstance(value, int):
             return datetime.datetime.fromtimestamp(value, pytz.utc)
         if isinstance(value, string_types):
+            if value == '0000-00-00 00:00:00':
```
Comment (ishirav) [inline src/infi/clickhouse_orm/fields.py]: Good catch, thanks!

File: README.rst [hunk 3]
```diff
@@ -228,8 +228,8 @@ You can create array fields containing any data type, for example::
     class SensorData(models.Model):
 
         date = fields.DateField()
-        temperatures = fields.ArrayField(fields.Float32Field)
-        humidity_levels = fields.ArrayField(fields.UInt8Field)
+        temperatures = fields.ArrayField(fields.Float32Field())
+        humidity_levels = fields.ArrayField(fields.UInt8Field())
```
Comment (ishirav) [inline README.rst]: You are correct, my bad

File: src/infi/clickhouse_orm/utils.py (hunk 1 above)
Comment (tsionyx) [inline src/infi/clickhouse_orm/utils.py]: But your implementation does not really work with the array of integers as it returned by ClickHouse:
```
>>> parse_array("[1,2,3]")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 18, in parse_array
IndexError: string index out of range

```

Comment (tsionyx) [inline src/infi/clickhouse_orm/utils.py]: just removed the ast-based parsing

**Pull request closed without merging**